test/e2e/storage/vsphere/vsphere_volume_vsan_policy.go

```diff
@@ -309,7 +307,9 @@ func invokeInvalidPolicyTestNeg(client clientset.Interface, namespace string, sc
 	return fmt.Errorf("Failure message: %+q", eventList.Items[0].Message)
 }
 
-func invokeStaleDummyVMTestWithStoragePolicy(client clientset.Interface, masterNode string, namespace string, clusterName string, scParameters map[string]string) {
+// invokeStaleDummyVMTestWithStoragePolicy assumes control plane node(aka. master node) is present on the datacenter specified in the workspace section of vsphere.conf file.
+// With in-tree VCP, when the volume is created using storage policy, shadow (dummy) VM is getting created and deleted to apply SPBM policy on the volume.
```

Comment:
AFAIK the word "dummy" can also be considered offensive, but given this is bound deeper in the test, you can leave it for now.

for reference there is a new WG that is being formed to clear some of this language, so i won't be surprised that this test is revisited in the near future:
https://github.com/kubernetes/community/pull/4884

alternatively you could replace the word "dummy" with something like "test" in this PR.

Reply:
Thanks, that is a nice point.
But if renaming this dummy to the other word like "test", it would be a little difficult to understand what that means because the function contains many "dummy"s.
So I guess it is nice to have another PRs to update such words for the other places.